Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/debugoptions/ui/DebugOptionsFragment.kt

```diff
@@ -49,11 +49,11 @@ class DebugOptionsFragment : Fragment(R.layout.fragment_test_debugoptions), Auto
                 if (!chip.isPressed) return@setOnCheckedChangeListener
                 val type = chip.text.toString()
                 vm.selectEnvironmentType(type)
-                MaterialAlertDialogBuilder(requireContext())
-                    .setTitle("Restarting ↻")
-                    .setMessage("Configuring $type environment. Get yourself a glass of water \uD83D\uDEB0")
-                    .setCancelable(false)
-                    .show()
+                displayDialog {
+                    setTitle("Restarting ↻")
+                    setMessage("Configuring $type environment. Get yourself a glass of water \uD83D\uDEB0")
+                    setCancelable(false)
```

Comment:
does `setCancelable`  here works ? should not you pass it in  displayDialog (cancelable=false)?

Reply:
Yep, you're right. I just copy pasted what was inside the Builder and didn't notice it.